### PR TITLE
Submetrizable

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by StevenClontz** [comment]
In addition to motivating the orphaned P112, we can now model question 34 of https://www.sciencedirect.com/science/article/pii/S0166864119300847

https://topology.pi-base.org/spaces?q=%24T_%7B3%20%5Cfrac%7B1%7D%7B2%7D%7D%24%2BStrategic%20Menger%2BHas%20a%20coarser%20separable%20metrizable%20topology%2B~%24%5Csigma%24-compact

File: properties/P000112.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-aliases: []
 name: Submetrizable
+aliases:
+- Has a metrizable compression
+- Submetrisable
 refs:
-- mr: MR2492954
-  name: ! 'The compact-open topology: A new perspective'
-counterexamples_id: null
+- doi: 10.2307/2042236
```
Comment (prabau) [inline properties/P000112.md:8]: 10.1090/S0002-9939-1978-0482639-9 seems a more accessible doi (directly to the AMS website)

File: properties/P000166.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+aliases:
+- Has a separable metrizable compression
+refs:
+- doi: 10.1007/BFb0098389
+  name: Topological Properties of Spaces of Continuous Functions
+---
+
+The topology contains a coarser topology which is both {P26} and {P53}.
+
+N.B. if the space is also {P6}, by {{doi:10.1007/BFb0098389}} this is equivalent to
```
Comment (prabau) [inline properties/P000166.md:13]: Is there a specific result we could point to from that reference?

**Comment by prabau** [comment]
A simple example of a space that is [has a coarser separable metrizable topology] + metrizable, but not separable:
S3 (discrete topology on the reals), as it has the Euclidean topology as a coarser one.

Somehow it's related to T404: if $X$ is sub-separable-metrizable, its coarser separable metrizable topology would imply by T404 that the space has cardinality at most the continuum.  So that's something we get for all sub-separable-metrizable spaces. (I don't think it's deducible, given the current infrastructure)

I think sub-separable-metrizable is a hereditary property, right?  So as a consequence, the discrete spaces that are sub-sep-metr as exactly those of cardinality <= continuum.

**Comment by prabau** [comment]
https://math.stackexchange.com/questions/4050041/how-did-submetrizability-come-into-existence-what-is-submetrizability-of-a-top mentions that Gruenhage has a survey article with a lot of results.

File: properties/P000166.md (hunk 2 above)
Comment (StevenClontz) [reply properties/P000166.md:13]: Ah I got away with this in my paper, but you're right, it should be pointed out more carefully. I'm on campus and hopefully can look this up...

Comment (StevenClontz) [reply properties/P000166.md:13]: It seems we have a hard copy but getting to the library isn't in scope for my day today. I'm going to open an issue to improve this when I can.

**Comment by StevenClontz** [comment]
More can be done for these submetrizable properties but perhaps this sufficient for one PR.

**Comment by prabau** [comment]
Yeah, we can add in later PRs.

**Review by prabau — APPROVED** [review]

**Pull request merged**